src/stories/layouts/Wizard/styles.scss

```diff
@@ -0,0 +1,71 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Wizard
+
+.wizard {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: $spacing-5;
```

Comment:
`grid-gap` is deprecated, I think `gap` should be used here

Reply:
Did not know that lol thanks!